### ISSUE TITLE
Word parts not supported by marker processor (context is unknown)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [brailleapps/dotify.api#22]
```
input word part markup e.g. <em>T</em>est.

This will be treated by the marker processor as a single word.

```

Original issue reported on code.google.com by `joel.hak...@mtm.se` on 9 Aug 2013 at 12:41

**Comment by joeha480** [comment]
Related to #29

**Comment by joeha480** [comment]
Fixed in dcba4d041272623a45073e997424d47096849938
